sign/ed25519/pubkey112.go

```diff
@@ -0,0 +1,20 @@
+// +build go1.12
```

Comment:
`go1.12` means Go 1.12 or later.  So we want to use `!go1.13` here, which effectively means Go 1.12 or earlier.